src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,547 @@
+                </TextContent>
+              }
+            >
+              <Button
+                variant="plain"
+                aria-label="About activation key"
+                className="pf-u-pl-sm pf-u-pt-0 pf-u-pb-0"
+                size="sm"
+              >
+                <HelpIcon />
+              </Button>
+            </Popover>
+          </TextListItem>
+          <TextListItem component={TextListItemVariants.dd}>
+            <ActivationKeyInformation />
+          </TextListItem>
+        </TextList>
+        <br />
+      </TextContent>
+      {isError && (
+        <Alert
+          title="Information about the activation key unavailable"
+          variant="danger"
+          isPlain
+          isInline
+        >
+          Information about the activation key cannot be loaded. Please check
+          the key was not removed and try again later.
+        </Alert>
+      )}
+    </>
+  );
+};
+
+export const ImageDetailsList = () => {
+  const blueprintName = useAppSelector((state) => selectBlueprintName(state));
+  const blueprintDescription = useAppSelector((state) =>
+    selectBlueprintDescription(state)
+  );
+
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        {blueprintName && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Image name
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintName}
+            </TextListItem>
+          </>
+        )}
+        {blueprintDescription && (
+          <>
+            <TextListItem
+              component={TextListItemVariants.dt}
+              className="pf-u-min-width"
+            >
+              Description
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {blueprintDescription}
+            </TextListItem>
+          </>
+        )}
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const OscapList = () => {
+  const oscapProfile = useAppSelector((state) => selectProfile(state));
+  const release = useAppSelector((state) => selectDistribution(state));
+  const { data } = useGetOscapCustomizationsQuery(
+    {
+      distribution: release,
+      // @ts-ignore if oscapProfile is undefined the query is going to get skipped, so it's safe here to ignore the linter here
+      profile: oscapProfile,
+    },
+    {
+      skip: !oscapProfile,
+    }
+  );
+
+  const enabledServices = data?.services?.enabled;
+  const enabledServicesDisplayString = enabledServices?.reduce(
+    (accumulator, currentValue) => accumulator + ' ' + currentValue,
+    ''
+  );
+  const disableServices = data?.services?.disabled;
+  const disableServicesDisplayString = disableServices?.reduce(
+    (accumulator, currentValue) => accumulator + ' ' + currentValue,
+    ''
+  );
```

Comment:
There seems to be an extra space at the beginning of the disabled/enabled services:

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/29e422cb-2689-4668-90d0-7476c302bead)

Reply:
Also the `.reduce` method seems quite heavy duty for this - could you maybe just use [join method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)? Something like this:

```typescript
  const enabledServicesDisplayString = data?.services?.enabled.join(' ');
  const disableServicesDisplayString = data?.services?.disabled.join(' ');
```

Reply:
simple with join, I like it but it seems that it is not fix the problem, maybe do you have another idea?

Reply:
its very strange its look like sometimes there is extra space and sometimes not